### PR TITLE
docs(Table): add sticky toolbar with table demo

### LINKED ISSUE #7425: Bug - [Composable Table] - [documentation enhancement - toolbar location if OuterScrollContainer is used]
**Describe the problem**
I have the need for a sticky header and a sticky toolbar, I think that is allowed with the design guidelines.
The documentation is not completely clear how to do that

My first try:
```
<PageSection hasOverflowScroll={true}>
            <Toolbar><ToolbarContent><ToolbarItem><Button variant="secondary">Hello</Button></ToolbarItem></ToolbarContent></Toolbar>
            <OuterScrollContainer>
                <InnerScrollContainer>
                    <TableComposable variant={"compact"} borders={true} isStriped isStickyHeader>
                         ....
```

This has the problem, that the table overflows the page section, so you scroll the table and the page section.
What appears to work correctly is

```
<PageSection hasOverflowScroll={true}>
        <OuterScrollContainer>
                <Toolbar><ToolbarContent><ToolbarItem><Button variant="secondary">Hello</Button></ToolbarItem></ToolbarContent></Toolbar>
                <InnerScrollContainer>
                    <TableComposable variant={"compact"} borders={true} isStriped isStickyHeader>
                         ....
```

If the toolbar is within the OuterScrollContainer there is no overflow of the page section, and scrolling seams to work correctly

At least if the window is height is big enough (although table is scrollin). There might be another issue now with some minimal height values,  at some point the table starts overflowing again

**How do you reproduce the problem?**
See problem description

**Expected behavior**
Better document the interaction of scrollable table, toolbars and page sections

**Is this issue blocking you?**
no, it's just not completely clear for a newbee how to get that working

**Screenshots**
outside_outerscrollcontainer_layout_always_wrong:
outside_outerscrollcontainer_layout_always_wrong

within_outerscrollcontainer_layout_correct_if_window_is_high_enough:
within_outerscrollcontainer_layout_correct_if_window_is_high_enough

within_outerscrollcontainer_there_might_be_a_minimum_height_issue_somewhere:
within_outerscrollcontainer_there_might_be_a_minimum_height_issue_somewhere

**What is your environment?**
 - OS: macos
 - Browser chrome, firefox
 - Version 2022.06
 -

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #7425

Adds a fullpage demo showing a sticky toolbar and table.

### CODE REVIEW

**Comment by kmcfaul** [comment]
Removed DashboardWrapper as I couldn't quite get the import to work in table - the JSX demos import it fine but the TSX has an issue with it.

**Review by gitdallas — APPROVED** [review]
https://patternfly-react-pr-7850.surge.sh/components/table/react-demos/sticky-columns-and-header-with-toolbar/

File: packages/react-table/src/demos/examples/ComposableTableStickyToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+                      </Td>
+                    </Tr>
+                  ))}
+                </Tbody>
+              </TableComposable>
+            </InnerScrollContainer>
+          </OuterScrollContainer>
+        </div>
+        <br />
+        Other content
```
Comment (tlabaj) [inline packages/react-table/src/demos/examples/ComposableTableStickyToolbar.tsx:250]: are these "Other content" line needed?

**Review by mcarrano — APPROVED** [review]
Looks good to me. Thanks @kmcfaul !

**Review by mattnolting — APPROVED** [review]
Nice work, LGTM!

File: packages/react-table/src/demos/examples/ComposableTableStickyToolbar.tsx (hunk 1 above)
Comment (kmcfaul) [reply packages/react-table/src/demos/examples/ComposableTableStickyToolbar.tsx:250]: I can remove them. I had added them initially to show the page scrolling but that may also be seen by making the window shorter.

**Review by tlabaj — APPROVED** [review]
LGTM!

**Pull request merged**